### DISCUSSION TITLE
Simple Internet Radio does not work with ESP32 board manager 3.0.3

### DISCUSSION THREAD

**Discussion by timtamneil** [schreibfaul1/ESP32-audioI2S#798] [Q&A]
I am using an ESP32-S3-WROOM-1 board and a PCM5102 in the simplest configuration. When I compile with Arduino-ESP32 version 2.0.17 the sketch runs perfectly.  However when I update the board to 3.0.3 all I get is a loud buzzing.

What I am I missing here? Any help would be appreciated

The sketch is as follows:

/*
This will only work when compiled with version 2.0.17 esp32 in board manager

*/
#include <Arduino.h>
#include <WiFi.h>
#include "Audio.h" //see my repository at github "https://github.com/schreibfaul1/ESP32-audioI2S"

#define I2S_DOUT      47
#define I2S_BCLK      48
#define I2S_LRC       45 

Audio audio;

String ssid =     "xxxxxx";
String password = "xxxxxx";

//**************************************************************************************************
//                                           S E T U P                                             *
//**************************************************************************************************
void setup() {
    
    Serial.begin(115200);
    
    WiFi.begin("xxxxxx", "xxxxxx");
    while (WiFi.status() != WL_CONNECTED){
        delay(2000);
        Serial.print(".");
    }
    Serial.println("Connect to xxxxxx");
    
    audio.setPinout(I2S_BCLK, I2S_LRC, I2S_DOUT);
    audio.setConnectionTimeout(1000, 1000);
    audio.setVolume(21); // 0...21
    audio.connecttohost("http://peacefulpiano.stream.publicradio.org/peacefulpiano.aac"); //  128k mp3

}
//**************************************************************************************************
//                                            L O O P                                              *
//**************************************************************************************************
void loop()
{
    audio.loop();
}
//**************************************************************************************************
//                                           E V E N T S                                           *
//**************************************************************************************************
void audio_info(const char *info){
    Serial.print("audio_info: "); Serial.println(info);
}

**Comment by schreibfaul1** [comment]
I don't think it has anything to do with V3.0.3. I have used a new AAC decoder, probably bytes were lost when writing to I2S. The "Peacefulpiano" Station now has been running for many minutes without any dropouts.

**Reply by timtamneil** [reply]
Thanks for your quick reply - where do I get the AAC decoder that you have used successfully?

**Comment by timtamneil** [comment]
The latest repository for  this library contains a very different aac_decoder.h, so it looks like the issue
was as you say and the latest decoder in the repository works
